### PR TITLE
fix REPL on 1.6

### PR DESCRIPTION
**Pull request by KristofferC**

### CODE REVIEW

File: src/repl.jl [hunk 1]
```diff
@@ -1,5 +1,14 @@
 
 promptname(level, name) = "$level|$name> "
+
+function write_prompt(terminal, mode)
+    @static if VERSION ≥ v"1.6.0-DEV.517"
```
Comment (timholy) [inline src/repl.jl:5]: Which PR is this? Sorry if I broke Debugger...

Comment (KristofferC) [reply src/repl.jl:5]: I don't think it was you. Some refactoring in how the REPL handled colors I think.

Comment (timholy) [reply src/repl.jl:5]: Sounds good. Still might be good to link to the PR if the DEV number indicates you know it.

Comment (timholy) [reply src/repl.jl:5]: No need to fix this PR, though, just for the future.

**Pull request merged**